governance/xc_admin/packages/xc_admin_common/src/chains.ts

```diff
@@ -19,6 +19,7 @@ export const RECEIVER_CHAINS = {
   aurora: 9,
   fantom: 10,
   celo: 14,
+  near: 15,
```

Comment:
well we are not using our own version of wormhole receiver here, right?

Reply:
No, I think we're using wormhole's own contract.

Reply:
well this one has the list of the ones that we manage the wh contract. (see the comment above)

Reply:
Moved it to a separate section with a comment.